circuitbreaker/circuit_breaker_test.go

```diff
@@ -45,20 +45,20 @@ func TestCircuitBreaker_ExecuteMultipleFallbacksFail(t *testing.T) {
 		ErrorPercentThreshold:  10,
 	})
 
-	circuitName := ""
+	circuitName := fmt.Sprintf("ExecuteMultipleFallbacksFail_%d", time.Now().Nanosecond()) // unique name to avoid conflicts with go tests `-count` option
```

Comment:
nit: the probabilities are probably low enough as it is, but we most likely want to use `uuid` for "truly unique" identifiers